src/appleseed-max-impl/appleseedoslplugin/osltexture.cpp

```diff
@@ -86,8 +86,9 @@ OSLTexture::OSLTexture(Class_ID class_id, OSLPluginClassDesc* class_desc)
     m_shader_info = m_class_desc->m_shader_info;
     m_texture_id_map = m_shader_info->m_texture_id_map;
 
-    m_has_uv_coords = m_shader_info->find_param("in_uvCoord") != nullptr;
-    m_has_xyz_coords = m_shader_info->find_param("in_placementMatrix") != nullptr;
+    m_has_uv_coords = m_shader_info->find_maya_attribute("uvCoord") != nullptr && 
```

Comment:
The following formatting is probably a bit easier to read:

    m_has_uv_coords =
        m_shader_info->find_maya_attribute("uvCoord") != nullptr && 
        m_shader_info->find_maya_attribute("uvFilterSize") != nullptr;